src/migrations/1627387313410_migrate-old-zonefiles.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+    await pgm.db.query(`
```

Comment:
I appreciate the proper sql migration here! However, I don't think we actually want a migration for this particular dataset given the number of zonefiles. Or, if we do merge with this migration, we'd probably want to "prune" it next time we making a sql-schema-breaking change. @rafaelcr thoughts?

Reply:
This PR would require a re-sync of all events to the DB, correct? If so, couldn't we just remove this migration and also remove the `zonefile` columns from the original `names` table migration?

Reply:
Ahh yes, @rafaelcr is right. At first I didn't notice the additional changes to previous existing migration files. Those are breaking sql changes, and there is no need for this extra migration file.